docs/cli.md

```diff
@@ -591,3 +596,29 @@ gcli group grant --group groupA --role admin
 ```bash
 gcli group revoke  --group groupA --role admin
 ```
+
+### Fileset commands
+
+#### Create a fileset
+
+```bash
+gcli fileset create --name hadoop.fileset.schema --fileset example --properties managed=true,location=file:/tmp/root/schema/example
+```
+
+#### List filesets
+
+```bash
+gcli fileset list --name hadoop.fileset.schema
```

Comment:
what's the "hadoop" mean? is it the metalake name or catalog name? Besides, this should be in the same style with table's command.

Reply:
it's a catalog name

Reply:
if "hadoop" is catalog, then "fileset" is the schema name, what the "schema" represents?